lib/broccoli/amd-shim.js

```diff
@@ -6,11 +6,14 @@ module.exports = function shimAmd(tree, nameMapping) {
     let name = nameMapping[relativePath];
     if (name) {
       return [
-        '(function(define){\n',
+        '(function(define, module){\n',
```

Comment:
@rwjblue Thanks for your feedback. I agree. The readability will increase when using templates. I will update my PR.